### PR TITLE
clarifying db host types

### LINKED ISSUE #187: Clarifying database Host_Types
There’s a lot of confusion about what the different database types are. Refined to clarify:

A&I_Database: Provides access to databases containing abstract and index information on scholarly articles intended to support discovery. 

Aggregated_Full_Content: Provides access to databases of full text serial and/or book content (monographs, reference works, etc.), and/or content otherwise aggregated into titles, where content is accessed in the context of the licensed database

Full_Content_Database: Provides access to databases that are a collection of content items that are not aggregated into titles (i.e. not part of a serial or book or other title). Full_Content_Database includes multimedia collections.

### PR DESCRIPTION
**Pull request by tashalouizamc**
per TAG discussions, updated definitions of aggregated_full_content and full_content_database host_types

### CODE REVIEW

File: source/appendices/a-glossary-of-terms.rst [hunk 1]
```diff
@@ -459,7 +459,7 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - Full_Content_Database
-     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not otherwise part of a serial or monograph (i.e. non-aggregated).
+     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not aggregated into titles (i.e. not part of a serial or book or other title).
```
Comment (berndoberknapp) [inline source/appendices/a-glossary-of-terms.rst]: Is it intentional that the "Full_Content_Database may include multimedia materials" isn't included here?

File: source/appendices/a-glossary-of-terms.rst [hunk 2]
```diff
@@ -459,7 +459,7 @@ Note: The main Code of Practice document takes precedence in the case of any con
...
 
        Note: In contrast to A&I_Databases and Aggregated_Full_Content the Investigations and Requests for Full_Content_Databases (like for example Cochrane Database of Systematic Reviews) are reported with Data_Type Database.
```
Comment (berndoberknapp) [inline source/appendices/a-glossary-of-terms.rst]: I assume the note about the Data_Type will be addressed in another pull request?

**Review by berndoberknapp — CHANGES_REQUESTED** [review]
I've removed the reference work changes in appendix A.

Please see the inline comments.

File: source/appendices/a-glossary-of-terms.rst (hunk 1 above)
Comment (tashalouizamc) [reply source/appendices/a-glossary-of-terms.rst]: It's a source of confusion and led to questions about when full content database should be used in place of multimedia collection.

**Review by tashalouizamc — COMMENTED** [review]
I can't approve changes but think this is ready to merge

File: source/appendices/a-glossary-of-terms.rst [hunk 3]
```diff
@@ -459,9 +459,7 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - Full_Content_Database
-     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not otherwise part of a serial or monograph (i.e. non-aggregated).
-
-       Note: In contrast to A&I_Databases and Aggregated_Full_Content the Investigations and Requests for Full_Content_Databases (like for example Cochrane Database of Systematic Reviews) are reported with Data_Type Database.
+     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not aggregated into titles (i.e. not part of a serial or book or other title). Full_Content_Database may include but not be exclusively composed of multimedia content items.
```
Comment (berndoberknapp) [inline source/appendices/a-glossary-of-terms.rst:462]: While reading the multimedia comment (which differs from the one above - this one seems better to me) I first thought that this would be wrong, but I've mixed Host_Types and Data_Types. A database with multimedia content only would indeed result in Host_Type Multimedia_Collection, but the Searches and Denial metrics would be reported with Data_Type Database_Full in DR.

**Review by berndoberknapp — APPROVED** [review]
